model/uzivatel.php

```diff
@@ -261,6 +261,49 @@ function jmeno()
     return $this->u['jmeno_uzivatele'].' '.$this->u['prijmeni_uzivatele'];
   }
 
+  /** Jméno uživatele */
+  function getJmeno()
+  {
+    return $this->u['jmeno_uzivatele'];
+  }
+  
+  /** Příjmení uživatele */
+  function getPrijmeni()
+  {
+    return $this->u['prijmeni_uzivatele'];
+  }
+  
+  /** Ulice a číslo popisné uživatele */
+  function getUlice()
+  {
+    return $this->u['ulice_a_cp_uzivatele'];
+  }
+  
+  /** Město uživatele */
+  function getMesto()
+  {
+    return $this->u['mesto_uzivatele'];
+  }
+  
+  /** Poštovní směrovací číslo uživatele */
+  function getPSC()
+  {
+    return $this->u['psc_uzivatele'];
+  }
+    
+  /** Číslo občanského průkazu uživatele */
+  function getOP()
+  {
+    return $this->u['op'];
+  }
+  
+  /** Číslo občanského průkazu uživatele */
+  function getDatumNarozeni()
+  {
+    $datum = new DateTimeCz($this->u['datum_narozeni']);
```

Comment:
Formátování bych nechal na view, vrátit ideálně objekt. Pokud s tím bude nějaký shit v šabloně, budem to řešit dál.

Reply:
asi je, nefunguje mi to. S touto částí mám trochu problém... nevím jak ji pojmout. Přehlíd jsem totiž, že už jedna metoda na datum existuje. Jenže ten tvůj hack v tom dělá trochu bordel, ale zase pokud tam není, zobrazují se nesmysly.